corehq/apps/data_interfaces/forms.py

```diff
@@ -288,6 +293,7 @@ class CaseRuleCriteriaForm(forms.Form):
     custom_match_definitions = forms.CharField(required=False, initial='[]')
     property_match_definitions = forms.CharField(required=False, initial='[]')
     filter_on_closed_parent = forms.CharField(required=False, initial='false')
+    ucr_filter_definitions = forms.JSONField(required=False, initial={})
```

Comment:
I don't think `initial` should be a mutable instance. And in `compute_initial()` this is a list.
```suggestion
    ucr_filter_definitions = forms.JSONField(required=False, initial=list)
```